### ISSUE TITLE
Error when add Instructions into Objective C project manually

### ISSUE THREAD

**Issue by minhthuc251** [ephread/Instructions#32]
I have a problem when i import Instructions into my project manually : 
<img width="1162" alt="screen shot 2016-03-05 at 11 12 44 am" src="https://cloud.githubusercontent.com/assets/5000594/13545588/9f18fe84-e2c3-11e5-81f5-daa5977b1b57.png">

and this if i create a new file swift and copy all code in instructionmanagement to new file.
<img width="1162" alt="screen shot 2016-03-05 at 11 16 22 am" src="https://cloud.githubusercontent.com/assets/5000594/13545594/d485dd94-e2c3-11e5-9f86-0a07fec72b32.png">

I dont know how to install Instrction by cocoapods with my current podfile.Can you help me.
<img width="560" alt="screen shot 2016-03-05 at 11 17 38 am" src="https://cloud.githubusercontent.com/assets/5000594/13545602/ff6a14bc-e2c3-11e5-8f87-27e29a4f363d.png">

**Comment by ephread** [comment]
Hello @minhthuc251!

Few things stand out :

1. the Framework doesn't seem to be properly linked, which method did you use to add it to your project?
2. iOS 8 is the lowest version supported by Instructions. You won't be able to make it work for iOS 6-7, which might lead to your first issue.

As for the Podfile you'd need to tell CocoaPods to use frameworks and to set the plateform, to at least, iOS 8. For instance:

```ruby
use_frameworks!

target 'EzMobile' do
    plateform :ios, '8.0'
    pod 'Instructions', '~> 0.4'
end
```

**Comment by minhthuc251** [comment]
1. I add instrutions by manually . But xcode always show error without i can cmd + click in InstructionManage and it move to InstructionManage class. 
2. If i use cocoapods like that : 

use_frameworks!

target 'iPadPOS' do
    plateform :ios, '8.0'
    pod 'Instructions', '~> 0.4'
    xcodeproj '....'
end

target 'iPadPOS' do
    plateform :ios, '6.0'
    pod 'Analytics/Segmentio'
    xcodeproj '....'
end

Is it true ?

**Comment by ephread** [comment]
1. What are the steps you followed to add the framework?
2. Supposedly so, although I'm not sure CocoaPods will accept two targets with the same name.

**Comment by AmeerTgh** [comment]
please add a sample in objective-c.

thanks

**Comment by minhthuc251** [comment]
My steps are : 
 1. I create a new swift file in my project to create objc briding header.
 2. Copy instructions.xcodeproj into my project.
 3. Add instructions framework into target.
 4. set Defines module and Embedded content contain Swift code and Install Objective-C compatibiliti header to YES in Build Setting.

I use @import Instructions and #import "iPadPOS-Swift.h" success but i can't invoke InstructionsManage with those xcode error. 
I really need use your library for my project. Please help me

**Comment by AmeerTgh** [comment]
@minhthuc251 
i created a sample for u & me ;)

[objective-c example.zip](https://github.com/ephread/Instructions/files/160993/objective-c.example.zip)

**Comment by ephread** [comment]
@AmeerTgh there is a rough example in the `objective-c` branch, that's the one @minhthuc251 is building upon. See #12.

@minhthuc251 It looks fine. I just recreated a new project, and performed the same steps you followed and… it works well. I don't know what to tell you, I have no clue about what's going on. For step 2, I copied both `Instructions.xcodeproj` and `Source/`, but I assume that's also what you did.
